src/javascript/esutil.ts

```diff
@@ -100,42 +101,41 @@ const VALID_EXPRESSION_TYPES = new Map([
  *
  * Accepts literal values, and native constructors.
  *
- * @param {Node} node An Espree expression node.
+ * @param {Node} node A Babel expression node.
  * @return {string} The type of that expression, in Closure terms.
  */
 export function closureType(
-    node: estree.Node, sourceRange: SourceRange, document: ParsedDocument):
+    node: babel.Node, sourceRange: SourceRange, document: ParsedDocument):
     string|Warning {
   const type = VALID_EXPRESSION_TYPES.get(node.type);
   if (type) {
     return type;
-  } else if (node.type === 'Literal') {
-    return typeof node.value;
-  } else if (node.type === 'Identifier') {
+  } else if (babel.isLiteral(node)) {
+    return typeof astValue.expressionToValue(node);
```

Comment:
this is kind of funny to me in it's original form... seems like this should be handled in `VALID_EXPRESSION_TYPES`

Reply:
I totally extended VALID_EXPRESSION_TYPES to do this now.